pkg/agent/controller/egress/egress_controller.go

```diff
@@ -196,6 +205,18 @@ func NewEgressController(
 	if trafficShapingEnabled && !openflow.OVSMetersAreSupported() {
 		klog.Info("EgressTrafficShaping feature gate is enabled, but it is ignored because OVS meters are not supported.")
 	}
+
+	eventBroadcaster := record.NewBroadcaster()
+	eventBroadcaster.StartStructuredLogging(0)
+	runtime.Must(scheme.AddToScheme(scheme2.Scheme))
+	eventBroadcaster.StartRecordingToSink(&v1.EventSinkImpl{
+		Interface: client.CoreV1().Events(""),
+	})
+	recorder := eventBroadcaster.NewRecorder(
+		scheme2.Scheme,
+		corev1.EventSource{Component: "antrea-agent"},
```

Comment:
I think the `Component` can be more specific to "egress-controller", @tnqn what's your thoughts?

Reply:
Yes, seems K8s is using specific component name, not process name.